stocks/historical.py

```diff
@@ -15,21 +16,43 @@ def data_ten_years_back_for_stock(request):
         ticker = body['ticker']
         stock = Stock(name=name, ticker=ticker)
         stock.save()
+        stock_id = stock.id
         now = arrow.now()
         ten_back = now.replace(years=-10)
         now = get_date_array_for_fetcher(now)
         ten_back = get_date_array_for_fetcher(ten_back)
         fetcher = Fetcher(ticker, ten_back, now)
         history = fetcher.getHistorical()
-        objects = []
-        for row in history.itertuples():
-            value = getattr(row, "Close")
-            date = getattr(row, "Date")
-            objects.append(
-                DailyStockQuote(value=value, date=date, stock=stock))
-        DailyStockQuote.objects.bulk_create(objects, batch_size=500)
+        save_stock_quote_from_fetcher(history, stock_id)
 
 
 def get_date_array_for_fetcher(arrow_date):
     """Function that formats arrow date to Yahoo Fetcher format """
     return [arrow_date.year, arrow_date.month, arrow_date.day]
+
+
+def fill_stocks(request):
+    """Function that fills stock data for missing days"""
+    if request.method == "POST":
+        data = DailyStockQuote.objects.values('stock__ticker', 'stock_id')\
+                .annotate(date=Max('date'))
+        for stock in data:
+            last_date = arrow.get(stock["date"]).replace(days=+1)
+            last_date = get_date_array_for_fetcher(last_date)
+            now = get_date_array_for_fetcher(arrow.now())
+            ticker = stock["stock__ticker"]
+            stock_id = stock['stock_id']
```

Comment:
Make variables like
STOCK_ID_FIELD = 'stock_id'
Instead of typing it out multiple times